src/content/en/tools/workbox/guides/configure-build-tools.md

```diff
@@ -0,0 +1,159 @@
+project_path: /web/tools/workbox/_project.yaml
+book_path: /web/tools/workbox/_book.yaml
+description: A guide on how to configure workbox-build, workbox-webpack-plugin, and workbox-cli.
```

Comment:
I would include the configuration in the relevant module pages (i.e. https://developers.google.com/web/tools/workbox/modules/workbox-cli) I know workbox-build and workbox-webpack don't have pages yet, but they need them and it would be the best place for this content. It would also drastically avoid having a mix of content for a range of tools.

If you would be ok with it, could we drop the IA changes and this doc, just land the _shared items and start adding workbox-build and workbox-webpack docs.